Gemfile

```diff
@@ -121,6 +121,7 @@ group :development, :test do
   gem 'haml-i18n-extractor'
   gem "active_merchant-paypal-bogus-gateway"
   gem 'rubocop', require: false
+  gem 'jasmine-rails'                   # testing for JavaScript code
```

Comment:
Wait, no, that only determines what gets installed, not what gets loaded at runtime. Bah.